Makefile

```diff
@@ -116,8 +120,8 @@ integration_test:
 
 coverage:
 	GO111MODULE=off go get github.com/zhouqiang-cl/gocovmerge
-	gocovmerge "$(TEST_DIR)"/cov.* | grep -vE ".*.pb.go|.*.__failpoint_binding__.go" > "$(TEST_DIR)/all_cov.out"
-	grep -vE ".*.pb.go|.*.__failpoint_binding__.go" $(TEST_DIR)/cov.unit_test.out > $(TEST_DIR)/unit_test.out
+	gocovmerge "$(TEST_DIR)"/cov.* | grep -vE ".*.pb.go|.*.__failpoint_binding__.go|dm/ctl/master/stop_relay.go" > "$(TEST_DIR)/all_cov.out"
```

Comment:
how about remove _stop_relay.go_ from the project directly? if needed, we can add it back.

Reply:
yep, removed